policy.go

```diff
@@ -0,0 +1,99 @@
+package judge
+
+import (
+	"errors"
+	"github.com/gearnode/judge/orn"
+	"strings"
+)
+
+// Policy is an entity in Judge that, when attached to an identity, defines
```

Comment:
You are using `identity` but never use this term in the code; is it equivalent to `entity`?